go.mod

```diff
@@ -24,7 +24,8 @@ require (
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
 	github.com/open-policy-agent/cert-controller v0.2.0
-	github.com/open-policy-agent/frameworks/constraint v0.0.0-20220218180203-c2a0d8cdf85a
+	github.com/open-policy-agent/frameworks/constraint v0.0.0-20220319020711-acd776435a23
+	github.com/open-policy-agent/opa v0.37.2
```

Comment:
Where does GK still reference opa directly? I thought everything is moving to framework?

Reply:
It looks like in two places (target and target test):

```
pkg/target/target.go:	"github.com/open-policy-agent/opa/storage"
pkg/target/target_test.go:	"github.com/open-policy-agent/opa/storage"
```

This is because the TargetHandler interface is currently expecting an OPA path for the storage key. IIRC this is b/c the path object has some formatting methods that are nice, but essentially it's a list of strings otherwise.

If we want to fully remove this dependency again, maybe a follow-on piece of work to remove this from the TargetHandler interface?

https://github.com/open-policy-agent/frameworks/blob/00eadf686a2876291b032335b6993dc41d770398/constraint/pkg/handler/handler.go#L19-L30

Reply:
ah now I remember. it makes sense to address this as a follow up work.